vms/secp256k1fx/camino_fx.go

```diff
@@ -268,3 +257,39 @@ func (fx *Fx) verifyMultisigUnorderedCredentials(tx UnsignedTx, creds []verify.V
 
 	return nil
 }
+
+// SplitPrivateKeys splits an array of PrivateKeys into `from` and `signers`
+// The delimiter is a `nil` PrivateKey.
+// If no delimiter exists, the given PrivateKeys are used for both from and signing
+// Having different sets of `from` and `signer` allow multisignatuew alias feature
+func SplitPrivateKeys(keys []*crypto.PrivateKeySECP256K1R) (set.Set[ids.ShortID], []*crypto.PrivateKeySECP256K1R) {
```

Comment:
Maybe renamed SplitPrivateKeys to something like SplitIntoSiginersAndFromAddrs ?
Cause split sounds very general - like split thing into many things
But this funcs does more specific task (and comments describe this specific)

In general, this concept of key array where part of keys are just as addrs feels to shady, but I guess thats done in order to do less avax changes

Reply:
Exactly I started some time ago with an additional parameter but it lead into hell on earth